### PR TITLE
feat(analyzer): getClassMemberDoc

### PR DESCRIPTION
**Pull request by bennypowers**
adds `getClassMemberDoc` helper

I'd like to use this in a suite of plugins I hope to publish shortly. Useful for things like adding async flag to methods, or readonly flag to fields

### CODE REVIEW

File: packages/analyzer/src/utils/manifest-helpers.js [hunk 1]
```diff
@@ -140,3 +140,25 @@ export function getModuleForClassLike(cem, className) {
...
+ * manifest doc for the module's class' member.
+ *
+ * @param  {Partial<import('custom-elements-manifest/schema').Module>} moduleDoc Manifest module
+ * @param  {string} className Class to get member of
+ * @param  {string} memberName Class member to get
+ * @return {import('custom-elements-manifest/schema').ClassMember|void} the requested class member
+ */
+export function getClassMemberDoc(moduleDoc, className, memberName) {
+  /** @type {import('custom-elements-manifest/schema').ClassDeclaration} */
+  const classDoc = (moduleDoc.declarations.find(x => x.name === className));
```
Comment (thepassle) [inline packages/analyzer/src/utils/manifest-helpers.js:154]: Note that this wont find classes that are exported as default

File: packages/analyzer/src/utils/manifest-helpers.js [hunk 2]
```diff
@@ -140,3 +140,25 @@ export function getModuleForClassLike(cem, className) {
...
+
+  if (!classDoc)
+    return console.warn(`Could not find class ${className}`);
+  if (!Array.isArray(classDoc.members))
```
Comment (thepassle) [inline packages/analyzer/src/utils/manifest-helpers.js:158]: we have a `has` helper [here](https://github.com/open-wc/custom-elements-manifest/blob/master/packages/analyzer/src/utils/index.js#L5) could you use that instead?

File: packages/analyzer/src/utils/manifest-helpers.js (hunk 1 above)
Comment (bennypowers) [reply packages/analyzer/src/utils/manifest-helpers.js:154]: should deal with that in https://github.com/webcomponents/custom-elements-manifest/issues/72, right?

File: packages/analyzer/src/utils/manifest-helpers.js (hunk 2 above)
Comment (bennypowers) [reply packages/analyzer/src/utils/manifest-helpers.js]: done

File: packages/analyzer/src/utils/manifest-helpers.js (hunk 1 above)
Comment (bennypowers) [reply packages/analyzer/src/utils/manifest-helpers.js:154]: also, with current spec, if user passes in `default` for `className`, then we're good, right?

Comment (thepassle) [reply packages/analyzer/src/utils/manifest-helpers.js:154]: Yeah thats true, just something to keep in mind 🙂

**Pull request merged**